e2e_testing/torchscript/elementwise.py

```diff
@@ -378,7 +380,31 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ElementwiseMulScalarModule())
 def ElementwiseMulScalarModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4))
-    
+
+
+
+class ElementwiseMulScalarPromotionModule(torch.nn.Module):
```

Comment:
`MulTensor => `MulScalar`test. 
`Promotion` => `TypePromotion` for consistency with the test below. IMO, we could remove `TypePromotion` because the core functionality of type promotion are tested in the type_promotion.py. The focus of these tests are checking the op itself is lowered correctly.

Reply:
Given that a test for these ops already exist, and a test for type promotion already exists, are these tests just redundant?

Reply:
hmm, I didn't find any mul.Tensor test but only a mul.Scalar test. Note, `torch.mul` could endup with different variant depending on the operand types. I think adding one test for `mul.Tensor` is good enough. Adding two tests for different types would be even better because Int and float are using different code paths.